### PR TITLE
Home task 1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ochubryk/add_task.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		<p align="center">выходной: <u>суббота</u>,<u>воскресенье</u></p>
+		<hr>
+		<h3>Вот такой текст</h3>
+		<h3><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h3>
+		<hr>
+		<h3>Акроним и курсив</h3>
+		<abbr title="Т.... Ф....">ТФ</abbr>: Y(x)=e<sup><i>iπx</i></sup>+log<sub><i>x</i></sub>4
+		<hr>
+		<h3>Тег small внтури абзаца</h3>
+		<p align="right"><small>Создано при помощи блокнота©</small></p>
```
Comment (pluhin) [inline ochubryk/add_task.html]: You did not close body and html tags

Comment (pluhin) [inline ochubryk/add_task.html]: Character "copyright" should be written as mnemonic

File: ochubryk/python03.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		print '%d * %d = %d ' % (num1,num2,num1*num2)
+		print '%d - %d = %d ' % (num1,num2,num1-num2)
+		print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)
+</pre>
+		<p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет что-
+		либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей
+		программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p>
```
Comment (pluhin) [inline ochubryk/python03.html]: Did not close body and html

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ochubryk/add_task.html (hunk 1 above)
Comment (ghost) [reply ochubryk/add_task.html]: Fixed

Comment (ghost) [reply ochubryk/add_task.html]: Fixed

File: ochubryk/python03.html (hunk 2 above)
Comment (ghost) [reply ochubryk/python03.html]: Fixed

File: ochubryk/add_task.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		<p align="center"><b>Магазин работает:</b></p>
+		<p align="center">c 08<u><sup>30</sup></u> до 18<u><sup>30</sup></u></p>
+		<p align="center">обед с 13<u><sup>00</sup></u> до 14<u><sup>00</sup></u></p>
+		<p align="center">выходной: <u>суббота</u>,<u>воскресенье</u></p>
+		<hr>
+		<h3>Вот такой текст</h3>
+		<h3><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h3>
+		<hr>
+		<h3>Акроним и курсив</h3>
+		<abbr title="Т.... Ф....">ТФ</abbr>: Y(x)=e<sup><i>iπx</i></sup>+log<sub><i>x</i></sub>4
```
Comment (pluhin) [inline ochubryk/add_task.html]: replace pi to mnemonic

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ochubryk/add_task.html (hunk 3 above)
Comment (ghost) [reply ochubryk/add_task.html]: Fixed

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**